### PR TITLE
Add Seldon Quickstart Docs

### PR DESCRIPTION
**Pull request by ukclivecox**
Signed-off-by: Clive Cox <cc@seldon.io>

Adds Quickstart Docs for Seldon.

### CODE REVIEW

**Comment by sriumcp** [comment]
@Clive Cox As part of your PR, can I request two additional (very minor) changes?

Search and replace `mkdocs serve -s` with `mkdocs serve` (you should see two search results for this).
Search and replace `mkdocs build -s` with `mkdocs build` (you should see one search result for this).

Silencing the strict option / mkdocs warnings seems necessary due to recent mkdocs deprecations.

I can confirm that this change works for me locally.

Sorry about requesting changes that are completely unrelated to your PR -- I will plan on pinning the version of `mkdocs` in the future so that this issue does not arise.

**Comment by clive** [comment]
I think you have the wrong person :)

File: mkdocs/docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -102,6 +104,12 @@ Choose the K8s stack over which you are performing the A/B testing experiment.
         ```shell
         $ITER8/samples/knative/quickstart/platformsetup.sh istio
         ```
+=== "Seldon"
+    Setup Seldon Core, Seldon Analytics and Iter8 within your cluster.
+
+    ```shell
+    $ITER8/samples/seldon/quickstart/platformsetup.sh
```
Comment (sriumcp) [inline mkdocs/docs/getting-started/quick-start.md:111]: I am looking into the `platformsetup.sh` more closely...

```shell
## 0(d). Ensure Helm is available
KUSTOMIZE_VERSION=$(helm  version | cut -d. -f2 | tail -c 2)
if [[ ${KUSTOMIZE_VERSION} -ge "3" ]]; then
    echo "Helm v3+ available"
else
    echo "Helm v3+ is unavailable"
    exit 1
fi
```

Could you please replace `KUSTOMIZE` above with `HELM`.

File: mkdocs/docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -756,6 +948,122 @@ Iter8 introduces a Kubernetes CRD called Metric that makes it easy to use metric
...
+          provider: prometheus
+          type: Gauge
+          urlTemplate: http://seldon-core-analytics-prometheus-seldon.seldon-system/api/v1/query
+        
+        ```
+
+
 
 ??? Note "Metrics in your environment"
     You can use metrics from any RESTful provider in Iter8 experiments.
```
Comment (sriumcp) [inline mkdocs/docs/getting-started/quick-start.md:1069]: Could  you please change this text  as follows?

`In this tutorial, the user-engagement metric is synthetically generated and served by mock New Relic service/Prometheus service`.

File: mkdocs/docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -1214,3 +1638,10 @@ When the experiment completes, you will see the experiment stage change from `Ru
     kubectl delete -f $ITER8/samples/knative/quickstart/experiment.yaml
     kubectl delete -f $ITER8/samples/knative/quickstart/experimentalservice.yaml
     ```
+
+=== "Seldon"
```
Comment (sriumcp) [inline mkdocs/docs/getting-started/quick-start.md:1642]: Please include `fortio.yaml` also in the cleanup.

File: mkdocs/docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -1163,6 +1543,50 @@ As the experiment progresses, you should eventually see that all of the objectiv
...
+                }
+              }
+            },
+            "weight": 75
+          }
+	
+        ```    
+
 
 As the experiment progresses, you should see traffic progressively shift from the baseline version to the candidate version.
```
Comment (sriumcp) [inline mkdocs/docs/getting-started/quick-start.md:1592]: > You created an Iter8 experiment with A/B testing pattern and progressive deployment pattern. In each iteration, Iter8 observed the latency and error-rate metrics collected by Prometheus, and the user-engagement metric from New Relic;

Could you please change New Relic to `New  Relic/Prometheus` above?

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor changes requested.

File: mkdocs/docs/getting-started/quick-start.md (hunk 1 above)
Comment (ukclivecox) [reply mkdocs/docs/getting-started/quick-start.md:111]: yes

File: mkdocs/docs/getting-started/quick-start.md (hunk 2 above)
Comment (ukclivecox) [reply mkdocs/docs/getting-started/quick-start.md:1069]: Isn't this text shown across all the stacks? In the Seldon case i'm not using mock NewRelic.

File: mkdocs/docs/getting-started/quick-start.md (hunk 3 above)
Comment (ukclivecox) [reply mkdocs/docs/getting-started/quick-start.md:1642]: yes

File: mkdocs/docs/getting-started/quick-start.md (hunk 4 above)
Comment (ukclivecox) [reply mkdocs/docs/getting-started/quick-start.md:1592]: yes

File: mkdocs/docs/getting-started/quick-start.md (hunk 2 above)
Comment (ukclivecox) [reply mkdocs/docs/getting-started/quick-start.md:1069]: Ah ok - understand now. Yes will change.

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**